src/Provider/DataSourceProvider.js

```diff
@@ -28,7 +27,7 @@ export function parseSourceData(data, extDest, layer) {
         mergeFeatures: layer.mergeFeatures === undefined ? true : layer.mergeFeatures,
         withNormal: layer.isGeometryLayer !== undefined,
         withAltitude: layer.isGeometryLayer !== undefined,
-        symbolToCircle: layer.symbolToCircle || false,
+        layers: source.layers,
```

Comment:
You could remove option `filter: layer.filter || source.filter` is previous option.

Reply:
you can verify but I thinks is ok

Reply:
No, for example it can be use like this https://github.com/iTowns/itowns/blob/master/examples/source_stream_wfs_25d.html#L245

Reply:
Good, so it's remove the ambiguity.